src/Model.cpp

```diff
@@ -461,11 +423,15 @@ QMap<int, GLuint> ModelCopyForRendering::render(QSharedPointer<Chain> chain) {
     // corresponding vertex's inputs
     QVector<QVector<int>> inputs;
 
-    auto & vao = chain->vao();
-    if(!vao.isCreated())
-        vao.create();
-    chain->setRealTime(timebase->wallTime());
-    chain->setBeatTime(timebase->beat());
+    auto vao = chain->vao();
+
+    // XXX
```

Comment:
This is totally fine; now `EffectNode` uses `Timebase` directly, which I think makes more sense.